crates/astria-sequencer-relayer/src/sequencer_relayer.rs

```diff
@@ -200,14 +253,18 @@ impl ShutDown {
         if let Some(mut api_task) = api_task {
             info!("sending shutdown signal to API server");
             let _ = api_shutdown_signal.send(());
-            match timeout(Duration::from_secs(4), &mut api_task)
+            let limit = Duration::from_secs(4);
+            match timeout(limit, &mut api_task)
                 .await
                 .map(crate::utils::flatten)
             {
-                Ok(Ok(())) => info!("api server exited gracefully"),
-                Ok(Err(error)) => error!(%error, "api server exited with an error"),
+                Ok(Ok(())) => info!("API server exited gracefully"),
+                Ok(Err(error)) => error!(%error, "API server exited with an error"),
                 Err(_) => {
-                    error!("api server did not shut down after 25 seconds; killing it");
```

Comment:
There was a copy-paste error here (`25` should have been `4`), so I extracted the value out to a variable.

I prefer having variable data separated from the main static message in `tracing` macros, so I did that here and also in the equivalent block above for the `relayer_task`.  Let me know if this is not a preference here please.